addons/account/models/account_move.py

```diff
@@ -6160,6 +6199,11 @@ def _get_available_invoice_template_pdf_report_ids(self):
 
         return available_reports
 
+    def _is_user_able_to_review(self):
+        if self.write_uid not in (self.env.user, self.env.ref('base.user_root')) or self.env.user != self.write_uid:
+            return self.env.user.has_group('account.group_account_user')
+        return self.write_uid.has_group('account.group_account_user')
```

Comment:
This doesn't seem right...
1. the cron user could be anyone
2. I think write_uid is always equal to self.env.user (at least in the cron case)
3. aren't we missing an ensure_one if you're accessing `write_uid.has_group` - EDIT: it's ok since has_group has it.
4. Does the `write_uid` even matter? Surely only the poster should. if the poster can check then it's checked.
why is `self.env.user.has_group('account.group_account_user')` not right?